jenkins/run

```diff
@@ -16,10 +16,6 @@ if [[ -z "${IMAGE_REPO}" ]]; then
     IMAGE_REPO="quay.io/pbench"
 fi
 
-# We delete any python compiler caches to avoid conflicts with the python compiler
-# in the container.
-find . -name __pycache__ -exec rm -r {} \; > /dev/null 2>&1
```

Comment:
If we want to remove this functionality all-together, great, let's open an issue and do that in a separate PR.  But the point of this commit is not to open that can of worms, but instead move this `find` command, unchanged, out of the `jenkins/run` context before `podman run` is invoked to the proper context in the container executed by `podman run`.

I did not want to investigate whether this is needed or not, just make it NOT hang forever in certain contexts.

Reply:
I don't understand why this move is necessary or how it solves the problem that you seem to think it solves.

Reply:
The `find` hangs when run from `jenkins/run`, which is before the container context because the `rm` command is asking to remove a file for which it does not have permission.  And yes, when you run this interactively, it still fails after answer yes.

Reply:
So...we can remove the files from inside the container which `jenkins/run` invokes, but we cannot remove the files from outside the container?  That's...hard to understand -- how can the container do something that its invoker cannot?

Reply:
yes, see `podman unshare`.

Reply:
There's something weird going on here that I'm not seeing.

We (meaning Mr. Jenkins) start out by checking out our sources from Git into what I naively think of as a local file system.  Later, but before running the tests, we apply the `find` command to this file system (via `jenkins/run`).  I would have naively thought that we're running `find` under the same UID (or sub-UID) as the one which ran the `git checkout`...so, there should be nothing which we cannot manipulate...right?

Now we invoke the container, things get potentially interesting, and perhaps we create a bunch of files (like the ones that we tried to delete earlier) which show up outside the container...and they probably are owned by some strange (sub-)UID.

And then perhaps Mr. Jenkins tries to clean these up and fails due to lack of permission?  And, perhaps these work areas are therefore left around?  And, perhaps a new test job, at some later time, tries to (re)use one of these work areas?  And, then it tries to run our favorite `find` command, which runs into terrible problems when it hits these leftover files to which it has no access without being inside the container or in a `podman unshare` session?

If that's the case, then we're trying to solve this problem in the wrong way!  We shouldn't be attacking it at the beginning of a test run, we should be doing it at the _end_ of the previous run.

That is, this seems like there is a lack in our Jenkins pipeline (like, maybe it should be operating under a `podman unshare` there?).